README.md

```diff
@@ -11,44 +14,78 @@ Computations and statistics on manifolds with geometric structures.
 - The documentation of ```geomstats``` can be found on the [documentation website](https://geomstats.github.io/).
 - If you find ``geomstats`` useful, please kindly cite our [paper](https://arxiv.org/abs/1805.08308).
 
+## Install geomstats via pip3
 
-## Installation
-
-OS X & Linux:
+From a terminal (OS X & Linux), you can install geomstats and its requirements with ``pip3`` as follows::
 
 ```
 pip3 install -r requirements.txt
 pip3 install geomstats
 ```
 
-Pytorch and tensorflow requirements are optional, as geomstats can be used with numpy only.
+This installs the latest version uploaded on PyPi.
+
+## Install geomstats via Git
+
+From a terminal (OS X & Linux), you can install geomstats and its requirements via Git as follows::
 
-To change backend:
 ```
-export GEOMSTATS_BACKEND=pytorch
+pip3 install -r requirements
+git clone https://github.com/geomstats/geomstats.git
 ```
 
-## Getting started
+This installs the latest GitHub version, useful for developers.
 
-Run example scripts, for example:
+## Choose the backend
+
+You can choose your backend by setting the environment variable ``GEOMSTATS_BACKEND`` to ``numpy``, ``tensorflow`` or ``pytorch``. By default, the numpy backend is used. You should only use the numpy backend for examples with visualizations.
+
+From the command line:
 
 ```
-python3 examples/plot_grid_h2.py
+export GEOMSTATS_BACKEND=pytorch
 ```
 
-## Contributing
-
-Developers can install the dev-requirements:
+and in the code:
 
 ```
-pip3 install -r dev-requirements.txt
+import geomstats.backend as gs
 ```
 
-And run unit tests from this folder:
+Note that pytorch and tensorflow requirements are optional, as geomstats can be used with numpy only.
+
+## Getting started
+
+To use ``geomstats`` for learning
+algorithms on Riemannian manifolds, you need to follow three steps:
+- instantiate the manifold of interest,
+- instantiate the learning algorithm of interest,
+- run the algorithm.
+
+The data should be represented by a ``gs.array``. This structure represents numpy arrays, or tensorflow/pytorch tensors, depending on the choice of backend.
+
+The following code snippet shows the use of tangent Principal Component Analysis on the
+space of 3D rotations, assuming ``data`` belongs to this space.
+
 ```
-nose2 tests
+from geomstats.geometry.special_orthogonal import SpecialOrthogonal
+from geomstats.learning.pca import TangentPCA
+
+so3 = SpecialOrthogonal(n=3)
+metric = so3.bi_invariant_metric
+
+tpca = TangentPCA(metric=metric, n_components=2)
+tpca = tpca.fit(data, base_point=metric.mean(data))
```

Comment:
data should be defined first:
`data = so3.random_uniform(10)`
Also, is this code old? `metric.mean(data)` won't work, just pass `tpca.fit(data)`

Reply:
Changed. Thanks!